library/tasks/SequenceDataWithMoleculeTagMetrics.wdl

```diff
@@ -0,0 +1,94 @@
+
+task CalculateGeneMetrics {
+  File bam_input
+  Int estimated_required_disk = ceil(size(bam_input, "G") * 2)
+
+  command {
+    set -e
+
+    CalculateGeneMetrics -i "${bam_input}" -o gene-metrics.csv
```

Comment:
is there a way the metrics collection tool can just output a gzipped file?

Reply:
Yes, that's a good idea.